doc/classes/PackedByteArray.xml

```diff
@@ -20,9 +20,9 @@
 			</description>
 		</method>
 		<method name="append">
-			<return type="void">
+			<return type="bool">
```

Comment:
How come `append` and `push_back` now return `bool`? It seems to match what is defined in `core/vector.h` but it returns `true` on failure and `false` on error, is that a weird Unix-style error handling?

Reply:
It's not an intended change but it comes from the C++ API. We now have to make sure that the C++ API matches what we want to expose to the scripting API.

Here I'm not sure there's any C++ code that relies on those return values, so they could possibly be changed to `void` to preserve the previous scripting API.